.github/workflows/mutation.yml

```
@@ -1,25 +1,25 @@
+      - "CHANGELOG.md"
+      - "docs/**"
+      - "README.md"
 
   push:
-    paths-ignore:
-      - 'docs/**'
-      - 'README.md'
-      - 'CHANGELOG.md'
-      - '.gitignore'
-      - '.gitattributes'
+    paths-ignore: *ignore-paths
```

actionlint (1.7.7)

"paths-ignore" section must be sequence node but got alias node with "" tag

(syntax-check)